apollo/manifest.json

```diff
@@ -10,7 +10,7 @@
   ],
   "manifest_version": "1.1.0",
   "maintainer": "sachin@apollographql.com",
-  "display_name": "Apollo Engine",
+  "display_name": "Apollo",
```

Comment:
Sorry, this cannot be changed, reverting in https://github.com/DataDog/integrations-extras/pull/728